cli/options.go

```diff
@@ -70,11 +70,52 @@ func WithName(name string) ProjectOptionsFn {
 // WithWorkingDirectory defines ProjectOptions' working directory
 func WithWorkingDirectory(wd string) ProjectOptionsFn {
 	return func(o *ProjectOptions) error {
+		if !filepath.IsAbs(wd) {
+			abs, err := filepath.Abs(wd)
```

Comment:
nit: any reason to test beforehand? Isn’t `filepath.Abs` a no-op if it’s already absolute?

Reply:
Actually it's not totally, on windows for example "/xx/yyy" would return true to `filepath.IsAbs()` but `filepath.Abs()` would update the path as a windows path, with drive etc. (always something fishy with windows paths 😁  )
That being said, maybe it's not an issue to still resolve `filepath.Abs()` regardless.